CMakeLists.txt

```diff
@@ -341,16 +342,20 @@ endif()
 
 
 #Add the various sub-projects
-add_subdirectory(abc)
+if(${ODIN_BUILD})
```

Comment:
it would make more sense for it all to be under the same if to keep things together as much as possible.

Also librtlnumber is only consumed by odin, so it should only be enabled if building with odin. 
blifexplorer depends on odin and this should also be reflected here.

I think ABC should have a separate flag i.e. WITH_ABC since odin and abc dont depend on each other and are discreet part of the flow

Reply:
> odin and abc

ODIN links against abc?

Reply:
I've updated the CMakeLists with you suggestions.